systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaRebalanceUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.utils.kafkaUtils;
+
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.resources.crd.KafkaRebalanceResource;
+import io.strimzi.test.TestUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import static io.strimzi.test.k8s.KubeClusterResource.kubeClient;
+
+public class KafkaRebalanceUtils {
+
+    private static final Logger LOGGER = LogManager.getLogger(KafkaRebalanceUtils.class);
+
+    private KafkaRebalanceUtils() {}
+
+    public enum KafkaRebalanceState {
+        PendingProposal,
+        ProposalReady,
+        Rebalancing,
+        Ready,
+        NotReady
```

Comment:
you are missing `Stopped`

Reply:
Done

Reply:
And that is why this should be part of the `api` ;-)